<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistoryDialog</class>
 <widget class="QDialog" name="HistoryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDateEdit" name="dateEdit_start">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>154</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_end">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>100</y>
     <width>154</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>501</width>
     <height>451</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
